.github/workflows/ci.yml

```diff
@@ -0,0 +1,503 @@
+name: ACS Packaging CI
+
+on:
+  pull_request:
+    branches:
+      - feature/**
+      - fix/**
+      - master
+      - release/**
+  push:
+    branches:
+      - feature/**
+      - fix/**
+  workflow_call:
+  workflow_dispatch:
+
+env:
+  AZURE_AUTHENTICATION_MODE: sharedKey
+  AZURE_STORAGE_ACCOUNT_NAME: acspackagingci
+  AZURE_STORAGE_ACCOUNT_KEY: ${{ secrets.AZURE_PACKAGING_STORAGE_ACCOUNT_KEY }}
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_EMAIL: ${{ secrets.BOT_GITHUB_EMAIL }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+  QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+  AWS_ACCESS_KEY: ${{ secrets.AWS_S3_PIPELINE_AMPS_ACCESS_KEY_ID }}
+  AWS_SECRET_KEY: ${{ secrets.AWS_S3_PIPELINE_AMPS_SECRET_ACCESS_KEY }}
+  TAS_ENVIRONMENT: ./tests/environment
+  TAS_SCRIPTS: ../alfresco-community-repo/packaging/tests/scripts
+  ALF_LICENCE_S3_PATH: s3://acs-license/acs/alf74-allenabled.lic
+  ALF_LICENCE_LOCAL_PATH: /tmp/licence.lic
+  PYTHON_VERSION: 3.7.15
+  DTAS_VERSION: v1.1
+
+jobs:
+  veracode:
+    name: "Source Clear Scan (SCA)"
+    runs-on: ubuntu-latest
+    if: >
+      ((github.ref_name == 'master' || contains(github.ref_name, 'release/')) && github.event_name != 'pull_request') &&
+      !contains(github.event.head_commit.message, '[skip tests]')
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.30.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.30.0
+      - name: "Init"
+        run: bash ./scripts/ci/init.sh
+      - uses: Alfresco/alfresco-build-tools/.github/actions/veracode@master
```

Comment:
We shouldn't use the `@master` ref, but the latest available tag instead.

Reply:
Thank you, I missed it! Updated to v1.30.0